services/movie.py

```diff
@@ -0,0 +1,33 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
```

Comment:
If argument can be `None`, you should show it in type hint
```suggestion
def get_movies(genres_ids: list[int] | None = None,
               actors_ids: list[int] | None= None) -> QuerySet:
```

Reply:
Fixed